packages/wallets/src/wallets.constants.ts

```diff
@@ -30,6 +31,7 @@ export interface NetworkConfig {
   tokenSymbol: string;
   rpcUrl: string;
   scan: string;
+  isDefaultNetwork?: boolean; //set true if the network exist in web3 extension.To set true fill in chainId,chainName,tokenSymbol
```

Comment:
If I'm correct that's redudant since this config used only as base. For example for as for metamask we have `isPopularNetwork()`.

Reply:
Cannot find usage

Reply:
you are right, i`ll remove it